mittab/urls.py

```diff
@@ -88,6 +88,8 @@
     url(r"^team/card/(\d+)/pretty/$",
         team_views.pretty_tab_card,
         name="pretty_tab_card"),
+    url(r"^tab_cards_json/$", team_views.tab_cards_json, name="tab_cards_json"),
```

Comment:
nit: would prefer `json` and `csv` be query params if possible, been a while since I touched a django app so not sure the easiest way to do that

Reply:
should this only be for csv and json, or should the DebateXML be combined as well? I imagine the tab cards webpage being made separate probably still makes sense though (file download vs webpage)